simulariumio/physicell/physicell_converter.py

```diff
@@ -6,12 +6,13 @@
 from pathlib import Path
 
 import numpy as np
-from .dep.pyMCDS import pyMCDS
+from ..dep.pyMCDS import pyMCDS
```

Comment:
is pyMCDS a physicell-specific dependency?

Reply:
yes it is. should I move it to that directory?